orchagent/intfsorch.cpp

```diff
@@ -61,24 +78,28 @@ void IntfsOrch::doTask(Consumer &consumer)
         if (op == SET_COMMAND)
         {
             /* Duplicate entry */
-            if (m_intfs.find(alias) != m_intfs.end() && m_intfs[alias].contains(ip_prefix.getIp()))
+            if (m_syncdIntfses.find(alias) != m_syncdIntfses.end() &&
+                m_syncdIntfses[alias].ip_addresses.contains(ip_prefix.getIp()))
             {
                 it = consumer.m_toSync.erase(it);
                 continue;
             }
 
             Port port;
-            if (!m_portsOrch->getPort(alias, port))
+            if (!gPortsOrch->getPort(alias, port))
             {
                 SWSS_LOG_ERROR("Failed to locate interface %s\n", alias.c_str());
-                it = consumer.m_toSync.erase(it);
+                it++;
                 continue;
             }
 
             if (!port.m_rif_id)
             {
                 addRouterIntfs(port);
-                m_intfs[alias] = IpAddresses();
+                IntfsEntry intfs_entry;
+                intfs_entry.ip_addresses = IpAddresses();
```

Comment:
ip_addresses is already initialized by default constructor. Why constructing another one and assigning?